src/DssSpell.sol

```diff
@@ -22,17 +22,46 @@ import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralOnboardingAction } from "./DssSpellCollateralOnboarding.sol";
 
+interface DssVestLike {
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function restrict(uint256) external;
+    function yank(uint256) external;
+}
+
 contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/5a8f09353e2a7d7f0d9f6a357cd05901348fe227/governance/votes/Executive%20vote%20-%20January%2024%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/<tbd>/2024%2C%202022.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-01-24 MakerDAO Emergency Executive Spell | Hash: 0x75da13d2bed7612e2e1960e7fe193453b66ead6e5d68b9a553258b0d421a21db";
+        "2022-01-28 MakerDAO Executive Spell | Hash: 0x";
+
+    address constant MCD_VEST_DAI          = 0x2Cc583c0AaCDaC9e23CB601fDA8F1A0c56Cdcb71;
+    address constant MCD_VEST_MKR_TREASURY = 0x6D635c8d08a1eA2F1687a5E46b666949c977B7dd;
```

Comment:
I fell in love with this, since we don't need to validate the address:
```suggestion
    address public immutable MCD_VEST_DAI = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
    address public immutable MCD_VEST_MKR_TREASURY = DssExecLib.getChangelogAddress("MCD_VEST_MKR_TREASURY");
```